src/stylesheets/app.scss

```diff
@@ -1,78 +1,77 @@
 // Global styles
-@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,700;1,100&display=swap");
-@import url("https://fonts.googleapis.com/css2?family=Average+Sans&display=swap");
-@import url("https://fonts.googleapis.com/css2?family=Hind+Madurai&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,700;1,100&display=swap');
```

Comment:
Probably not something to fix in this PR, but i feel like these imports should be in `index.scss`, considering that's where all the other imports are

Reply:
Yep, the changes are just showing here because of the linting